SmartDeviceLink-iOS.xcodeproj/project.pbxproj

```diff
@@ -8913,13 +9208,16 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				DEVELOPMENT_TEAM = NCVC2MHU7M;
+				CODE_SIGN_STYLE = Manual;
+				CURRENT_PROJECT_VERSION = 4;
+				DEVELOPMENT_TEAM = G99VP5HXV7;
```

Comment:
You do not need to commit changes to this file; these changes seem specific for your environment